CMakeLists.txt

```diff
@@ -47,7 +47,7 @@ set(Boost_LIBRARY_DIRS /usr/lib64/boost169)
 # Daq rpm variables
 set(Daq_VERSION 4.2.3)
 set(Daq_URL "${LSST_REPO}/daq/R4-V2.3.tgz")
-set(Daq_DIR "/srv/nfs/lsst-daq/daq-sdk/current")
+set(Daq_DIR "/opt/lsst/daq")
```

Comment:
Sorry, we cannot change this. It has to be in this location. One way to get around this is download the tar.gz file from lsst-web/~hwin16 and unpack here and make sure it looks the same.